update_changelog.sh

```diff
@@ -19,13 +19,13 @@ features_for() {
 
 gen_changelog() {
   {
-  echo "# Changelog"
+  echo "# Release notes"
   for REL in $(releases);
   do
     echo -e "\n## $REL" ;
     features_for $REL
   done
-  } > changelog.md
+} > docs/latest_release_notes.md
```

Comment:
As this contains all the releases, let's not call it "latest"... and now that I took a closer look, I'm not sure if "release notes" is a better name than "changelog": the generated content looks more like a changelog than full blown release notes, no?

Reply:
I'm good with changing it back to changelog and moving on.

Reply:
I withdraw that last comment.  The old docsify layout says Latest Release Notes. I say we keep it and argue the minutia later.

This note did cause me to notice that I forgot to add the page to the table of contents so thank you! :)